File: Knapsack-Problem/Item.cs
namespace KnapsackProblem {
    internal class Item(int value, int weight, int index) {
        public int value { get; set; } = value;
        public int weight { get; set; } = weight;
        public int index { get; set; } = index;

        public override string ToString() {
            return "no." +  index + "\tv: " + value + "\tw:" + weight;
        }
    }
}

File: Java/Knapsack-Problem/src/Item.java
public class Item {
    public int weight;
    public int value;
    public int index;

    public Item(int value, int weight, int index) {
        this.weight = weight;
        this.value = value;
        this.index = index;
    }

    @Override
    public String toString() {
        return "Item " + index +": value = " + value + ", weight = " + weight;
    }
}
